docs/src/main/sphinx/release/release-392.md

```diff
@@ -0,0 +1,70 @@
+# Release 392 (2 August 2022)
+
+## General
+
+* Add support for dynamic filtering with task retries. ({issue}`9935`)
+* Add support for correlated sub-queries for ``DELETE`` operations. ({issue}`9447`) 
+* Fix potential query failure in certain complex queries with multiple joins and
+  aggregations. ({issue}`13315`)
+
+## JDBC driver
+
+* Add the ``assumeLiteralUnderscoreInMetadataCallsForNonConformingClients``
+  configuration property as a replacement for
+  ``assumeLiteralNamesInMetadataCallsForNonConformingClients``, which is
+  deprecated and planned to be removed in a future release. ({issue}`12761`)
+* Add report for total time elapsed reading data from JDBC sources. ({issue}`13132`)
+
+## Delta Lake connector
+
+* Add support for using a randomized location when creating a table, so that
+  future table renames or drops do not interfere with new tables created with
+  the same name. ({issue}`12980`)
+* Add ``delta.metadata.live-files.cache-ttl`` configuration property for the
+  caching duration of active data files. ({issue}`13316`)
+* Add retention of metadata properties after schema changes. ({issue}`13368`)
+* Prevent writing to a table with `NOT NULL` or
+  [invariants](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#column-invariants)
+  columns. ({issue}`12635`)
+* Fix incorrect min and max statistics when writing `NULL`. ({issue}`13389`)
+
+## Druid connector
+
+* Add support for timestamp predicate pushdown. ({issue}`8404`)
+* Change mapping for the Druid `float` type to the Trino `real` type instead of
+  the `double` type. ({issue}`13412`)
+
+## Hive connector
+
+* Add support for short timezone IDs when translating Hive views. For example,
+  `JST` now works as an alias for `Asia/Tokyo`. ({issue}`13179`)
+* Add support for Amazon S3 Select pushdown for JSON files. ({issue}`13354`)
+
+## Iceberg connector
+
+* Add support for hidden `$file_modified_time` columns. ({issue}`13082`)
+* Add support for the Avro file format. ({issue}`12125`)
+* Improve query performance for tables with updated or deleted rows. ({issue}`13092`)
+* Improve performance of the `expire_snapshots` command for tables with many
+  snapshots. ({issue}`13399`)
+* Use unique table locations by default. ({issue}`12941`)
```

Comment:
We can mention that this can be disabled by setting `iceberg.unique-table-location=false`.